cli/cycpp.py

```diff
@@ -746,6 +748,7 @@ def includeloc(self, statement=None):
         'std::list': ('T',),
         'std::pair': ('T1', 'T2'),
         'std::map': ('Key', 'T'),
+        'std::tuple': ('T',),
```

Comment:
Or you could use some stuff in itertools to fake an infinite list. That is probably better.